mapiproxy/modules/mpm_cache.c

```diff
@@ -386,14 +387,14 @@ static NTSTATUS cache_pull_OpenMessage(struct dcesrv_call_state *dce_call,
 
 	message = talloc((TALLOC_CTX *)mpm, struct mpm_message);
 	NT_STATUS_HAVE_NO_MEMORY(message);
-	
-	message->session = mpm_session_init((TALLOC_CTX *)mpm, dce_call);
+
+	message->session = mpm_session_init(dce_call, NULL);
 	NT_STATUS_HAVE_NO_MEMORY(message->session);
```

Comment:
I'd love to have a link to an established policy for OpenChange but you already know there is no such written document. However we have hundreds of thousands of lines of code and this line looks nothing similar to anything we have :-)